src/app/api/users/route.ts

```diff
@@ -2,14 +2,20 @@ import { NextResponse } from 'next/server'
 import { cookies } from 'next/headers'
 import { createServerClient } from '@/utils/supabase'
 
-export async function GET() {
+export async function GET(request: Request) {
   const cookieStore = cookies()
   const supabase = createServerClient(cookieStore)
 
-  const { data, error } = await supabase.from('Users').select()
+  const url = new URL(request.url)
+  const userId = url.searchParams.get('userId')
+
+  const { data, error } = await supabase.from('users').select().eq('id', userId)
```

Comment:
I'd recommend selecting only the columns you need.  Later on this table might have things like the last time someone logged in and security settings and you don't want to send that out to the client unless needed by the front end.

Reply:
Added so that it only selects the name column from the users table.

Reply:
@ZhenyuYu1  - this change doesn't look like it selects the name column.  Do you have a commit you forgot to add?